cmd/scheduler/app/scheduler.go

```diff
@@ -160,7 +160,7 @@ func run(opts *options.Options, stopChan <-chan struct{}, registryOptions ...Opt
 		return fmt.Errorf("unable to get hostname: %v", err)
 	}
 	// add a uniquifier so that two processes on the same host don't accidentally both become active
-	id := hostname + "_" + uuid.New().String()
+	id := hostname + "_" + string(uuid.NewUUID())
```

Comment:
Unify uuid repo to `k8s.io/apimachinery/pkg/util/uuid` instead of `github.com/google/uuid`.

https://github.com/kubernetes/kubernetes/blob/765ce8a5ec49da009c44ce49e9b7ef47c3c0b005/cmd/kube-controller-manager/app/controllermanager.go#L248

Reply:
I don't think unifying UUID repo is necessary. But no big deal.